modules/nf-core/custom/tabulartogseacls/environment.yml

```diff
@@ -2,4 +2,9 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - conda-forge::coreutils=8.30
+  - conda-forge::coreutils=9.5
```

Comment:
We should not be pinning thise many dependencies in the environment.yml, this is not a lockfile.